run_wml_tests

```diff
@@ -159,19 +184,25 @@ class WesnothRunner:
             print(res.stdout.decode('utf-8'))
             if self.verbose > 1:
                 print("Result:", res.returncode)
+        num_passed = res.stdout.count(b"PASS TEST")
+        test_summary.pass_test(num_passed)
```

Comment:
I just realized this won't work for tests that are _expected_ to fail... posting to remind me to fix it.